.github/workflows/qa-env-on-pull-request.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   heroku-pull-request:
```

Comment:
Thanks again Berkeli for explaining the inner workings of this yml

Reply:
you are most welcome